.github/workflows/ci.yml

```diff
@@ -44,6 +44,63 @@ jobs:
     - run: npm run lint
     - run: npm run flow
     - run: npm test -- --maxWorkers=2 --ci
+    - name: Build Linux Packages
+      if: runner.os == 'Linux'
+      run: |
+        set -x
+        node scripts/release --linux
+    - name: Persist rpm package
+      if: runner.os == 'Linux'
+      uses: actions/upload-artifact@v1
+      with:
+        name: linux-release
+        path: ./dist/installers
+
+  integration_test_centos:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        node-version: [12.x]
+    container:
+      image: centos:8
+      env:
+        # This environment variable causes chrome to run with no sandbox- needed
+        # whenever chrome is run as root.
+        APPVEYOR: true
+    needs: build
+    steps:
+    - run: dnf install -y xorg-x11-server-Xvfb git nss-tools pulseaudio
```

Comment:
Please file an issue to have us create our own centos8 image that contains this software.

Reply:
Sure. Do we have a repo where these images are created? Are they attached to ci?

Reply:
When we had circleci, a dockerfile was put in the .circleci directory, and the image the dockerfile created was used in CI. So far we only use others' images. I don't believe a separate repository is needed.

Reply:
Er, I was assuming you meant Git repository. We have a brimsec org on docker hub where a built image can be uploaded.

Reply:
#701